### PR TITLE
automat: bump to 20.2.0 and drop -27

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

File: components/python/automat/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -25,6 +25,8 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 file path=usr/bin/automat-visualize
 file path=usr/bin/automat-visualize-2.7
 file path=usr/bin/automat-visualize-3.5
```
Comment (AndWac) [inline components/python/automat/manifests/sample-manifest.p5m:26]: Something looks wrong here. You have removed Python-2.7 support so I would expect all related files to be removed from the sample-manifest, too.

File: components/python/automat/test/results-3.5-32.master [hunk 2]
```diff
@@ -0,0 +1,5 @@
+_test/test_core.py .....                                                 [  8%]
+_test/test_discover.py .......................                           [ 45%]
+_test/test_methodical.py .................                               [ 73%]
```
Comment (AndWac) [inline components/python/automat/test/results-3.5-32.master:3]: running the tests here gives
-_test/test_discover.py .......................                           [ 45%]
+_test/test_discover.py sssssssssssssssssssssss                           [ 45%]

File: components/python/automat/manifests/sample-manifest.p5m (hunk 1 above)
Comment (alarcher) [reply components/python/automat/manifests/sample-manifest.p5m:26]: Probably I forgot to clobber... damn I hate that we do not clean the prototype before each install.

File: components/python/automat/test/results-3.5-32.master (hunk 2 above)
Comment (alarcher) [reply components/python/automat/test/results-3.5-32.master:3]: Who knows which other python modules these tests use... :S

File: components/python/automat/manifests/sample-manifest.p5m (hunk 1 above)
Comment (alarcher) [reply components/python/automat/manifests/sample-manifest.p5m:26]: FIxed.

**Pull request merged**